docs/docker-integration/index.rst

```diff
@@ -0,0 +1,92 @@
+.. _docker-integration:
+
+======================
+Docker, Swarm, Compose
+======================
+
+Flocker works with Docker, Swarm and/or Compose via the Flocker Plugin for Docker.
```

Comment:
"Flocker integrates with Docker Swarm via the Flcoker Plugin for Docker"

OR

perhaps that sentence can be deleted.